docs/js-core-1/module-project/index.md

```diff
@@ -0,0 +1,9 @@
+---
+id: index
+title: JS1 Module Project
+sidebar_label: Overview
+---
+
+## Overview
+
+The JS1 Module Project will appear here in due time...
```

Comment:
Let's maybe remove this, and the references to it in the homework, until that time

Reply:
removed the links